src/Aws/Services/UserDynamoDbStorageService.cs

```diff
@@ -108,12 +114,33 @@ public async Task<List<InternalUser>> GetAllUsers()
 
     public async Task<InternalUser> UpdateUser(InternalUser user)
     {
+        var s3Key = "unknown.jpg";
```

Comment:
May be worth having a comment explaining what unknown.jpg is and why we're uploading it.